projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/radio-button-docs/radio-button-docs.component.html

```diff
@@ -74,6 +74,47 @@ <h4 class="go-heading-4 go-heading--underlined">Code</h4>
         </div>
       </div>
 
+      <div class="go-column go-column--100">
```

Comment:
@Ashygoyal what do you think about moving this documentation into its own card down by the theme example? That way we can create a link to it and it is a little easier to find on the page.

Reply:
Agreed, fixed in latest commit.